source/backends/debian/debpkg.d

```diff
@@ -56,7 +58,16 @@ public:
     @property override const(string[string]) description () const { return desc; }
 
     override
-    @property string filename () const { return debFname; }
+    @property string filename () const {
+        if (debFname.isRemote) {
+            immutable path = buildPath (tmpDir, debFname.baseName);
+
+            downloadFile (debFname, path);
```

Comment:
Shouldn't that be `downloadIfNecessary`? Otherwise we might unnecessarily download multiple times...